src/main/java/org/coffeemine/app/spring/components/HoursLogIn.java

```diff
@@ -0,0 +1,58 @@
+package org.coffeemine.app.spring.components;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.button.ButtonVariant;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.dialog.Dialog;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.notification.Notification;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import org.coffeemine.app.spring.data.Task;
+import java.time.LocalDate;
+
+public class HoursLogIn {
+    private Task newTask;
+    private Dialog dialog;
+    private FormLayout newForm;
+    private Button hoursLogIn;
+    private Button create;
+    private Button cancel;
+
+    public HoursLogIn(){
```

Comment:
I mean, good effort but this is not how we do things ;-;
lesson of today: do not blindly copy someones' work and always question code quality.

Reply:
Yes sure, I asked chen his code on Friday and thought his code is clear for me to understand. so I didn't even rename attributes and method.  But I should bear in mind next time.

Reply:
His code is clear only after he explains to you.
The quality of Chen's code is unarguably not decent enough